packages/supabase-auth-helpers-qwik/src/index.ts

```diff
@@ -1,5 +1,7 @@
 // Methods
-export { createBrowserClient, createServerClient } from './utils/createSupabaseClient';
+// export { createBrowserClient, createServerClient } from './utils/createSupabaseClient';
```

Comment:
don't leave code commented unless it's used during debugging/maintenance. Just delete.